### ISSUE TITLE
PNPM audit detects low impact security vulnerability for @kubb/plugin-oas@2.18.9

### ISSUE THREAD

**Issue by kopach** [kubb-labs/kubb#1014]
### What is the problem this feature would solve?

run `pnpm audit` in project having `@kubb/plugin-oas@2.18.9` as a dependency

here is result

```
┌─────────────────────┬────────────────────────────────────────────────────────┐
│ low                 │ es5-ext vulnerable to Regular Expression Denial of     │
│                     │ Service in `function#copy` and                         │
│                     │ `function#toStringTokens`                              │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Package             │ es5-ext                                                │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Vulnerable versions │ >=0.10.0 <0.10.63                                      │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Patched versions    │ >=0.10.63                                              │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Paths               │ . > @kubb/plugin-oas@2.18.9 > @kubb/oas@2.18.9 >       │
│                     │ oas@24.3.5 > memoizee@0.4.15 > d@1.0.1 >               │
│                     │ es5-ext@0.10.62 > es6-iterator@2.0.3 > es5-ext@0.10.62 │
│                     │                                                        │
│                     │ . > @kubb/plugin-oas@2.18.9 > @kubb/oas@2.18.9 >       │
│                     │ oas@24.3.5 > memoizee@0.4.15 > es5-ext@0.10.62 >       │
│                     │ es6-iterator@2.0.3 > d@1.0.1 > es5-ext@0.10.62         │
│                     │                                                        │
│                     │ . > @kubb/plugin-oas@2.18.9 > @kubb/oas@2.18.9 >       │
│                     │ oas@24.3.5 > memoizee@0.4.15 > es5-ext@0.10.62 >       │
│                     │ es6-iterator@2.0.3 > es5-ext@0.10.62                   │
│                     │                                                        │
│                     │ ... Found 54 paths, run `pnpm why es5-ext` for more    │
│                     │ information                                            │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ More info           │ https://github.com/advisories/GHSA-4gmj-3p3h-gm8h      │
└─────────────────────┴────────────────────────────────────────────────────────┘
```

### External documents/projects?

- https://github.com/advisories/GHSA-4gmj-3p3h-gm8h
- https://github.com/medikoo/memoizee/issues/133

### What is the feature you are proposing to solve the problem?

please, update dependencies if possible

### What alternatives have you considered?

You can also patch the dependency with `pnpm` which should not bundle vulnerable dependency into published package which is already better that nothing, but that most probably won't stop scanners from reporting this I believe as scanners do usually parse only main sections of `package.json`

**Comment by kopach** [comment]
@stijnvanhulle, note, fixed and released as v0.4.16 in `memoizee`. So, just update of `memoizee` will fix the issue
https://github.com/medikoo/memoizee/issues/133#issuecomment-2127945114

**Comment by stijnvanhulle** [comment]
@kopach Will be fixed when https://github.com/readmeio/oas/pull/876 is merged

**Comment by stijnvanhulle** [comment]
@kopach It should be resolved with `v2.19.4`
